cmd/cnbBuild.go

```diff
@@ -339,6 +340,14 @@ func addConfigTelemetryData(utils cnbutils.BuildUtils, data *cnbBuildTelemetryDa
 	data.BuildTool = buildTool
 
 	data.Buildpacks.FromConfig = privacy.FilterBuildpacks(config.Buildpacks)
+
+	dockerImage, err := getDockerImageValue("cnbBuild")
+	if err != nil {
+		log.Entry().Warnf("Retrieving docker image failed: '%v'", err)
```

Comment:
```suggestion
		log.Entry().Warnf("Error while preparing telemetry: retrieving docker image failed: '%v'", err)
```

Should we add more context to this warning?